e2e/utils.go

```diff
@@ -975,6 +974,14 @@ func validatePVCSnapshot(
 			name := fmt.Sprintf("%s%d", f.UniqueName, n)
 			p.Spec.DataSource.Name = name
 			wgErrs[n] = createPVCAndApp(name, f, &p, &a, deployTimeout)
+			if dataPool != "" {
```

Comment:
compare with `noDataPool` here

Reply:
@nixpanic  As far as I think, putting the condition like dataPool != noDataPool might be a little confusing; as we are checking if data pool is empty, and comparing it being not equal to no data pool. wdyt?